### DISCUSSION TITLE
Environment Variable does not work

### DISCUSSION THREAD

**Discussion by iniacor** [vercel/community#2884] [Help] [answered]
### Summary

Hi! I set Environment Variable here https://vercel.com/iniacor/powercode-test/settings/environment-variables , but when I send data from the form in the console I get the message Uncaught (in promise) SyntaxError: Unexpected token '<', "<br />
<b>"... is not valid JSON. Without setting the Environment Variable, it works fine (when I specify the data directly in the application code in send_mail.php). Please help me understand what is the reason. my repo -  https://github.com/iniacor/powercode-test

### Steps to Reproduce

Fill data in form inputs, and click on submit button.

**Comment by smaeda-ks** [comment, marked as answer]
@iniacor 

> Fatal error: Uncaught Error: Undefined constant "process" in /var/task/user/api/send_mail.php:13 Stack trace: #0 {main} thrown in /var/task/user/api/send_mail.php on line 13

`process.env` is not the right PHP syntax to read environment variables. Please check the PHP documentation:
https://www.php.net/manual/en/reserved.variables.environment.php

**Reply by iniacor** [reply]
Thanks again!
